synfig-studio/src/gui/docks/dock_info.cpp

```diff
@@ -213,7 +214,12 @@ void studio::Dock_Info::on_stop_button_clicked()
 		Gtk::MESSAGE_QUESTION,
 		_("Cancel"),
 		_("Stop")))
-		async_renderer->user_stop();
+	{
+		sigc::bind(
+			sigc::mem_fun(*async_renderer, &studio::AsyncRenderer::stop),
+			studio::AsyncRenderer::INTERACTION_BY_USER);
```

Comment:
This is useless :) You're not using it: `sigc::bind` is used (as argument) for `signal_xxx.connect()` ;)

Reply:
ok, this is used only in case of a `signal` connection.